src/Resources/config/services/command.xml

```diff
@@ -15,7 +15,6 @@
             <argument type="service" id="setono_sylius_image.provider.processable_resource"/>
             <argument type="service" id="setono_sylius_image.command_bus"/>
             <argument type="service" id="setono_sylius_image.config.variant_collection"/>
-            <argument type="service" id="event_dispatcher"/>
```

Comment:
We still wanted to fire an event inside this command, right?

Reply:
Hmm, yep - will correct